Building ruby artifacts is broken on Linux

https://grpc-testing.appspot.com/view/Artifacts/job/gRPC_build_artifacts/361/architecture=x64,language=ruby,platform=linux/console

```
src/core/iomgr/pollset_set_windows.c: In function 'grpc_pollset_set_create':
src/core/iomgr/pollset_set_windows.c:40:19: error: type of 'pollset_set' defaults to 'int' [-Werror]
 grpc_pollset_set* grpc_pollset_set_create(pollset_set) { return NULL; }
                   ^
src/core/iomgr/pollset_set_windows.c:40:1: error: number of arguments doesn't match prototype
 grpc_pollset_set* grpc_pollset_set_create(pollset_set) { return NULL; }
 ^
In file included from ./src/core/iomgr/pollset_set_windows.h:37:0,
                 from src/core/iomgr/pollset_set_windows.c:38:
./src/core/iomgr/pollset_set.h:46:19: error: prototype declaration
 grpc_pollset_set *grpc_pollset_set_create(void);
                   ^
cc1: all warnings being treated as errors
make: *** [/tmp/objs/opt/src/core/iomgr/pollset_set_windows.o] Error 1
```

--- a/src/core/iomgr/pollset_set_windows.c
+++ b/src/core/iomgr/pollset_set_windows.c
@@ -37,7 +37,7 @@
 
 #include "src/core/iomgr/pollset_set_windows.h"
 
-grpc_pollset_set* grpc_pollset_set_create(pollset_set) { return NULL; }
+grpc_pollset_set* grpc_pollset_set_create(void) { return NULL; }
 
 void grpc_pollset_set_destroy(grpc_pollset_set* pollset_set) {}
 

